src/pages/en/guides/integrations-guide/cloudflare.md

```diff
@@ -88,14 +88,26 @@ $ pnpm install wrangler --save-dev
 
 It's then possible to update the preview script in your `package.json` to `"preview": "wrangler pages dev ./dist"`.This will allow you run your entire application locally with [Wrangler](https://github.com/cloudflare/wrangler2), which supports secrets, environment variables, KV namespaces, Durable Objects and [all other supported Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/#adding-bindings).
 
+## Access to the Cloudflare runtime
+
+You can access all the Cloudflare bindings and environment variables from Astro components and API routes through the adapter API.
+
+    import { getRuntime } from "@astrojs/cloudflare/runtime";
+
+    getRuntime(Astro.request);
```

Comment:
This needs actual code block backticks around it — backtick-less code blocks aren’t supported in Astro-flavoured Markdown. Will need fixing in the [Cloudflare README](https://github.com/withastro/astro/blob/main/packages/integrations/cloudflare/README.md).

Reply:
Will be fixed by: https://github.com/withastro/astro/pull/5210